### ISSUE TITLE
Make it editable

### ISSUE THREAD

**Issue by antonmoiseev** [PolymerElements/paper-dropdown-menu#100]
According to [the spec](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons), dropdown button can be editable. Are there any plans to implement this functionality? 
![image](https://cloud.githubusercontent.com/assets/182853/11727657/30ba82ca-9f96-11e5-96a0-5492e7ccc7a8.png)

**Comment by miztroh** [comment]
+1

**Comment by cdata** [comment]
Yes, but it would probably be implemented as a different element.

**Comment by marcelduran** [comment]
@cdata, why a different element? Why not a boolean attribute that makes `paper-dropdown-menu` editable? FWIW, just removing `readonly` from underlying `paper-input` "pretty much" does the job.

**Comment by davebaol** [comment]
+1

**Comment by Gwalks91** [comment]
+1

**Comment by eitch** [comment]
This issue is still open, is there an estimated ETA, or will this not be implemented?

**Comment by DomNomNom** [comment]
+1
Feature is still wanted.

**Comment by mweggen** [comment]
+1

**Comment by dwalters-eip** [comment]
+2

**Comment by Ghostfly** [comment]
+3

**Comment by RemiLelaidier** [comment]
+4
